charts/linkerd2/templates/linkerd-values.yaml

```diff
@@ -0,0 +1,33 @@
+{{ if or (.Values.tracing.enabled) -}}
```

Comment:
Seems like the `or` here is not needed?
Also if tracing is not enabled then `$dupValues` won't be defined. Will that cause a problem down at the end of this file?

Reply:
This if condition is to not have `linkerd-values` cm to be deployed, when there are zero add-ons enabled. Hence, the `or` condition but once we make the `linkerd-values` common for all config, then we should be able to remove this if condition.